src/utils/allowedHTML.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Defines the list of allowed HTML tags that do not count towards the total
+ * character limit.
+ *
+ */
+const regrexTests = [
+  new RegExp("(<iframe.*/iframe>)", "gm"),
+  new RegExp("(<div.*/div>)", "gm"),
```

Comment:
could i ask why the regexp is not `<iframe>.*</iframe>` (technically still the same but a lil clearer) and why we are including the parenthesis in the regexp?

Reply:
Sure, it was to allow for attributes for the <iframe> element. Eg. <iframe src="abc.xyz"></iframe> will not be matched!

Reply:
would `<iframe .*>.*</iframe>` be clearer here?